charts/jiralert/ci/test-values.yaml

```diff
@@ -5,3 +5,6 @@ config:
   receivers:
     - name: 'default'
       project: EXAMPLE
+
+ingress:
+  enabled: true
```

Comment:
Why enabled by default?

Reply:
Recheck the context. This is the test values file and ingress should be covered by tests